modules/installation-custom-gcp-vpc.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-vpc.adoc
+
+:_content-type: CONCEPT
+[id="installation-custom-gcp-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into existing subnets in an existing Virtual Private Cloud (VPC) in Google Cloud Platform (GCP). By deploying {product-title} into an existing GCP VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option. You must configure networking for the subnets.
+
+[id="installation-custom-gcp-vpc-requirements_{context}"]
+== Requirements for using your VPC
+
+The VPC’s CIDR block and the machine network CIDR must be non-empty. The subnets must be within the machine network.
```

Comment:
Per Matthew, the truth is "**the union of** the VPC CIDR and the machine network CIDR must be non-empty", rather than both **VPC's CIDR block** AND **machine network CIDR** are non-empty.

Reply:
@jianli-wei Addressed